src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
+In this exemple, the [Description] tag is set as step test case in the allure-reporter tool.  
+The [AllureTms] tag contains the test case name in Jama.
+
+Use the following example as a basis for your test cases.
+
+```C#
+    [TestFixture("ACR", "mg/mmol")]
+    [TestFixture("Chol/HDL", "")]
+    [AllureNUnit]
+    public class QCAfinionTestCases : UnitTestBase<AfinionAnalyzer>
+    {
+        string code;
+        string unit;
+        string flag;
+
+        XDocument translatedResult = null;
+        OBSR01 observationMessage = null;
+        HELR01 helloMessage = null;
+
+        public QCAfinionTestCases(string code, string unit)
+        {
+            this.code = code;
+            this.unit = unit;
+        }
+
+        [AllureEpic("TestAfinion")]
+        [AllureTms("When the device has a control result for upload, Then the message shall be upload to GWP")]
+        [AllureFeature("SYN-TC-2643 - The scope of this scenario is to verify that the translator returns the message as expected. QC results are shown in GEM Web Plus.")]
+        [Description("Check that the mapping for ACR is correct.")]
```

Comment:
The [Description] must contain the action to perform. We recommend not to use the word "Check that.." in the description tag.
The expressions "Check that, verify that..." must be part of the Expected Result, managed as a [Step] in the JSON.

Reply:
In this case, the description acts as a step, because the Step case does not work.

Reply:
Yes, we recommend to use terminology related with the interaction with the SUT, not related with the verification process, so in this case you could use what you are using in JAMA, "Send the message xxx" as action and "The message is translated as expected."